script.js

```diff
@@ -0,0 +1,205 @@
+let inputText = document.getElementById("input-name");
+let greetText = document.getElementById("greeting");
+const nameForm = document.getElementById("add-name-form");
+let userPet = {};
+let sideNav = document.getElementById("mySidenav");
+let fitMeter = document.getElementById("health-meter");
+let ageMeter = document.getElementById("age-meter");
+let hungerMeter = document.getElementById("hunger-meter");
+let petStatus = document.getElementById("pet-text");
+let gameContainer = document.getElementById("game-container");
+
+class Pet {
+  constructor(name) {
+    this.name = name;
+    this.age = 0;
+    this.hunger = 0;
+    this.fitness = 10;
+    this.isAlive = true;
+    this.isAdult = false;
+    this.isHappy = true;
+  }
+  deathCheck() {
+    if (this.fitness <= 0) {
+      console.log("pet died due to poor health");
```

Comment:
Do you need the console logs if you are already displaying the relevant message? Generally console logs should be removed, their usual purpose is for debugging